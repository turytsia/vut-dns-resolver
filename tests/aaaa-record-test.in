-r -t -6 -s kazi.fit.vutbr.cz www.google.com