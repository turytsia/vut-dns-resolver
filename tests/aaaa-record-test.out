Authoritative: No, Recursive: Yes, Truncated: No
Question section (1)
 www.google.com., AAAA, IN
Answer section (1)
 www.google.com., AAAA, IN, 0, 2a00:1450:4014:80f::2004
Authority section (0)
Additional section (0)
